tests/test_public_api.py

```diff
@@ -34,15 +34,15 @@ def test_hypotest(backend, model_setup):
 
 
 def test_prob_models(backend):
-    tb, _ = backend
+    tb, _ = pyhf.get_backend()
     pyhf.probability.Poisson(tb.astensor([10.0])).log_prob(tb.astensor(2.0))
     pyhf.probability.Normal(tb.astensor([10.0]), tb.astensor([1])).log_prob(
         tb.astensor(2.0)
     )
 
 
 def test_pdf_batched(backend):
-    tb, _ = backend
+    tb, _ = pyhf.get_backend()
```

Comment:
this should not have been changed. We have to inject the backend this way for testing purposes (if the tests are in weird orders, the `pyhf.get_backend()` isn't properly reset for tests).